tests/test_deployments/conftest.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
```

Comment:
is it customary / acceptable / good practice to have nested conftest files? I simply don't know but I guess so since that is where `autouse=True` goes to look for fixtures.